numpy/core/tests/test_regression.py

```diff
@@ -457,7 +457,7 @@ def test_swap_real(self, level=rlevel):
 
     def test_object_array_from_list(self, level=rlevel):
         # Ticket #270
-        np.array([1, 'A', None])  # Should succeed
```

Comment:
It does succeed - I just wanted it to make an assertion.
Happy to revert though.